app/helpers/halala.rb

```diff
@@ -0,0 +1,25 @@
+# coding: utf-8
+# frozen_string_literal: true
+class Halala
+  CURRENCY= ' SAR'
+  attr_reader :riyal
```

Comment:
I'll make it private for now, if we need it later, we can wrap its functionality with tests